coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 87.5
+    "covered_percent": 84.44
```

Comment:
My guess is that the files I recently deleted (mailer preview and application mailer) had a part in the coverage.